### ISSUE TITLE
Enhance active cells/rows

### ISSUE THREAD

**Issue by ttowntom** [NoroffFEU/yeetsheet#97]
**Stop resize**
Activating a cell resizes it's row. Make sure the row does not resize when a cell is active.

**Text color in active cell, dark mode**
Due to accessibility reasons the text color of active text in dark mode should be changed to uphold WCAG compatability.
